android/src/com/mygdx/game/Analysis/Detection.java

```diff
@@ -0,0 +1,147 @@
+package com.mygdx.game.Analysis;
+
+import android.os.Debug;
+
+import java.util.Set;
+import java.util.Map;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.ArrayList;
+
+import com.mygdx.game.DebugLog;
+/**
+ * This class handles checking the vector data for certain FUTURE: Actions
+ * and FUTURE: Patterns. For now these are just single functions with descriptive names.
+ * TODO: Implement more generic framework.
+ */
+public class Detection {
+    /**
+     * Constructor, initializes class fields.
+     * @param data Data interface object.
+     */
+    public Detection(final Data data) {
+        this.data = data;
+    }
+
+    /**
+     * Checks if the person's hands can not be found for a given amount of time.
+     * @param dt Given amount of time for it to be true.
+     * @return if the Action was detected or not.
+     */
+    public boolean handsFound(float dt) {
+        boolean inAction = false;
+        float action_time = 0;
+        for(int i = 0; i < data.getFrameCount(); i++) {
+            if(action_time >= dt) {
+                return true;
+            }
+            Vec2 r_handCoord = data.getCoord(i, body_part.r_wrist);
+            Vec2 l_handCoord = data.getCoord(i, body_part.l_wrist);
+            if(!inAction) {
+                if(r_handCoord.isNull() &&
+                        l_handCoord.isNull()) {
+                    action_time = 0;
+                    inAction = true;
+                }
+            }
+            if(inAction) {
+                if(!r_handCoord.isNull()  &&
+                        !l_handCoord.isNull()) {
```

Comment:
isNull returnt wat? en dat flip je? 

xD

Reply:
Oude code, je moet wel up-to-date changes reviewen D:

Reply:
:'(